Rn entry 2025 08.0 (#5632)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/release-notes/2025-okta-identity-engine/index.md

## August

### Monthly release 2025.08.0

| Change | Expected in Preview Orgs |
|--------|--------------------------|
| [Associated Domain Customizations API is self-service EA in Preview](#associated-domain-customizations-api-is-self-service-ea-in-preview) | July 16, 2025 |
| [Automate SCIM Integration for OIN Apps with Express Configuration](#automate-scim-integration-for-oin-apps-with-express-configuration) | August 7, 2025 |
| [Breached Credentials Protection is GA in Preview](#breached-credentials-protection-is-ga-in-preview) | May 15, 2025 |
| [Clear user factors for all devices is GA in Preview](#clear-user-factors-for-all-devices-is-ga-in-preview) | August 7, 2025 |
| [Cross App Access is self-service EA in Preview](#cross-app-access-is-self-service-ea-in-preview) | August 7, 2025 |
| [Custom FIDO2 AAGUID is self-service EA in Preview](#custom-fido2-aaguid-is-self-service-ea-in-preview) | July 16, 2025 |
| [Device signal collection policy is self-service EA in Preview](#device-signal-collection-policy-is-self-service-ea-in-preview) | August 7, 2025 |
| [Encryption of ID tokens and access tokens is EA](#encryption-of-id-tokens-and-access-tokens-is-ea) | August 7, 2025 |

> | [Expanded use of user.getGroups() function in Okta Expression Language is GA in Production](#expanded-use-of-usergetgroups-function-in-okta-expression-language-is-ga-in-production) |June 4, 2025 |

Reviewer: ```suggestion
| [Expanded use of user.getGroups() function in Okta Expression Language is GA in Production](#expanded-use-of-user-getgroups-function-in-okta-expression-language-is-ga-in-production) |June 4, 2025 |
```

| [Granular configuration for Keep Me Signed In is GA in Preview](#granular-configuration-for-keep-me-signed-in-is-ga-in-preview) | January 8, 2025 |
| [Group Push Mappings is GA in Preview](#group-push-mappings-is-ga-in-preview) | August 7, 2025 |

> | [MyAccount Password API is GA in Production](#myaccount-password-api-is-ga-in-production) | July 2, 2025 |

Reviewer 2: ```suggestion
| [MyAccount Password API is GA in Production](#myaccount-password-api-is-ga-in-production) | May 24, 2023 |
```

| [MyAccount WebAuthn API is EA in Production and GA in Preview](#myaccount-webauthn-api-is-ea-in-production-and-ga-in-preview) | July 16, 2025 |
| [Multiple active IdP signing certificates is EA](#multiple-active-idp-signing-certificates-is-ea) | August 7, 2025 |

> | [New User Authenticator Enrollments API is self-service EA in Preview](#new-user-authenticator-enrollments-api-is-self-service-ea-in-preview) | July 9, 2025|

Reviewer 2: ```suggestion
| [New User Authenticator Enrollments API is GA in Preview](#new-user-authenticator-enrollments-api-is-ga-in-preview) | July 9, 2025|
```

| [New user profile permission](#new-user-profile-permission) | August 7, 2025 |

> | [OAuth 2.0 provisioning for Org2Org with Autorotation is GA in Production](#oauth-20-provisioning-for-org2org-with-autorotation-is-ga-in-production) | July 2, 2025 |

Reviewer 3: ```suggestion
| [OAuth 2.0 provisioning for Org2Org with Autorotation is GA in Production](#oauth-2-0-provisioning-for-org2org-with-autorotation-is-ga-in-production) | July 2, 2025 |
```

| [Universal Logout in the OIN Wizard](#universal-logout-in-the-oin-wizard) | August 7, 2025 |
| [Web app integrations now mandate the use of the Authorization Code flow](#web-app-integrations-now-mandate-the-use-of-the-authorization-code-flow) | August 7, 2025 |
| [Developer documentation updates in 2025.08.0](#developer-documentation-updates-in-2025080) | August 7, 2025 |
| [Bug fixed in 2025.08.0 ](#bug-fixed-in-2025-08-0) | August 7, 2025 |

#### Associated Domain Customizations API is self-service EA in Preview

You can now use the [Associated Domain Customizations API](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/AssociatedDomainCustomizations/) to view and update the associated domains of your custom domain. Associated domains let you build a trust relationship among your app, the referring domain, the user's credentials that are associated with that domain, and your brand in Okta. This feature makes it easier to adopt phishing-resistant authenticators, like passkeys in the FIDO2 (WebAuthn) authenticator.

> See [Customize associated domains](https://developer.okta.com/docs/guides/custom-well-known-uri/main/). <!-- ASSOCIATED_DOMAIN_CUSTOMIZATION OKTA-935558 -->

Reviewer 2: ```suggestion
See [Customize associated domains](/docs/guides/custom-well-known-uri/main/). <!-- ASSOCIATED_DOMAIN_CUSTOMIZATION OKTA-935558 -->
```

Express Configuration is a feature designed to automate the setup of SSO and SCIM for instances of OIN SaaS integrations by enterprise customers with minimal manual effort. It allows enterprise customers to securely configure OIDC and SCIM integrations without copying and pasting configuration values between Okta and Auth0-enabled apps. See [Express Configuration](https://developer.okta.com/docs/guides/enable-express-configuration/SCIM/main/). <!-- EXPRESS_CONFIGURATION_SCIM_FOR_AUTH0_APPS OKTA-977761 -->

#### Breached Credentials Protection is GA in Preview

Protect your org from the impact of credentials that have been compromised. If Okta determines that a username and password combination has been compromised after being compared to a third-party curated dataset, the protection response is customizable through password policies, including resetting the user's password, forcing a logout, or calling a delegated Workflow. See the [Okta Policies API](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/Policy/). <!-- BREACHED_CREDENTIALS_ADMIN_UX OKTA-822284 -->

#### Clear user factors for all devices is GA in Preview

> When you make a Revoke all user sessions (/api/v1/users/{userId}/sessions) request, include the new [`forgetDevices` parameter](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/UserSessions/#tag/UserSessions/operation/revokeUserSessions!in=query&path=forgetDevices&t=request) to clear a user's remembered factors on all devices. The user is prompted for full authentication on their next sign-in attempt from any device. The `forgetDevices` parameter is set to `true` by default. <!-- ENG_ENABLE_FORGET_DEVICES_INTENT_FOR_DELETE_USER_SESSIONS_API OKTA-979587 -- >

Reviewer: ```suggestion
When you make a Revoke all user sessions (/api/v1/users/{userId}/sessions) request, include the new [`forgetDevices` parameter](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/UserSessions/#tag/UserSessions/operation/revokeUserSessions!in=query&path=forgetDevices&t=request) to clear a user's remembered factors on all devices. The user is prompted for full authentication on their next sign-in attempt from any device. The `forgetDevices` parameter is set to `true` by default. <!-- ENG_ENABLE_FORGET_DEVICES_INTENT_FOR_DELETE_USER_SESSIONS_API OKTA-979587 -->
```

#### Cross App Access is self-service EA in Preview

> Admins can now manage third-party app data sharing with the new Cross App Access feature in the Okta Admin Console. This feature moves complex consent processes away from end-users, enhancing security and streamlining the experience. Once configured, end users can access their data from other SaaS apps without navigating OAuth consent flows. See the [Applications Cross Apps Connections API](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/ApplicationCrossAppAccessConnections/) and [Configure Cross App Access](https://help.okta.com/oie/en-us/content/topics/apps/apps-cross-app-access.htm). <!-- ENABLE_CONNECT_WITH_OKTA OKTA-800616) -- >

Reviewer: ```suggestion
Admins can now manage third-party app data sharing with the new Cross App Access feature in the Okta Admin Console. This feature moves complex consent processes away from end-users, enhancing security and streamlining the experience. Once configured, end users can access their data from other SaaS apps without navigating OAuth consent flows. See the [Applications Cross Apps Connections API](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/ApplicationCrossAppAccessConnections/) and [Configure Cross App Access](https://help.okta.com/oie/en-us/content/topics/apps/apps-cross-app-access.htm). <!-- ENABLE_CONNECT_WITH_OKTA OKTA-800616) -->
```

#### MyAccount WebAuthn API is EA in Production and GA in Preview

You can now use the [MyAccount WebAuthn API](https://developer.okta.com/docs/api/openapi/okta-myaccount/myaccount/tag/WebAuthn/) to enroll, list, and delete WebAuthn enrollments. Admins can build out an in-app passkey creation and enrollment experience with the MyAccount WebAuthn API operations. <!-- IDP_MY_ACCOUNT_API_WEBAUTHN OKTA-971173 -->

#### Multiple active IdP signing certificates is EA

Okta now supports multiple active signing certificates for a single SAML identity provider (IdP), enabling seamless certificate rotation with zero downtime. Use the new [`additionalKids` parameter](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/IdentityProvider/#tag/IdentityProvider/operation/createIdentityProvider!path=protocol/0/credentials/trust/additionalKids&t=request) to add another signing certificate for the IdP. You can upload up to two certificates per IdP connection. This improvement eliminates the need for tightly coordinated swaps with IdP partners and reduces the risk of authentication failures due to expired certificates. <!-- IDP_ENABLE_MULTIPLE_SIGNING_CERTS OKTA-986239 -->

> #### New User Authenticator Enrollments API is self-service EA in Preview

Reviewer 2: ```suggestion
#### New User Authenticator Enrollments API is GA in Preview
```

#### Universal Logout in the OIN Wizard

Universal Logout (UL) in the Okta Integration Network Wizard allows you to build, test, and submit UL functionality to the Okta Integration Network (OIN). Universal Logout lets you terminate users' sessions and revoke their tokens for supported OIN apps, as well as for generic OpenID Connect (OIDC) and Security Assertion Markup Language (SAML) apps. See [Submit an integration with the OIN Wizard](https://developer.okta.com/docs/guides/submit-oin-app/openidconnect/main/). <!-- UL_SUBMISSION OKTA-821575 -->

#### Web app integrations now mandate the use of the Authorization Code flow

To enhance security, web app integrations now mandate the use of the Authorization Code flow, as the Implicit flow is no longer recommended. See [Build a Single Sign-On (SSO) integration](https://developer.okta.com/docs/guides/build-sso-integration/openidconnect/main/#determine-the-oauth-2-0-flow-to-use).
<!-- OKTA-703909 -->

#### Developer documentation updates in 2025.08.0

> * The [Archived Okta Identity Governance API changelog (2023-2024)](/docs/release-notes/oig-changelog/) has been removed. For updates on these APIs, see the Okta Identity Governance API release notes.

Reviewer 4: ```suggestion
* The **Archived Okta Identity Governance API changelog (2023-2024)** has been removed. For updates on these APIs, see the Okta Identity Governance API release notes.
```
